docs/visual-testing/integrations/csharp.md

```diff
@@ -335,6 +336,44 @@ VisualClient = VisualClient.Create(Driver, Region.UsWest1, sauceUsername, sauceA
 VisualClient.CaptureDom = true;
 ```
 
+### Full page screenshots
+
+By default, only the current viewport is captured when `.VisualCheck` is used. You can opt in to capturing the entire page by using the `FullPage` option. It will capture everything by scrolling and stitching multiple screenshots together.
+Additionally, you have the option to configure full page settings using the `FullPageConfig` option.
+
+:::note
+It's recommended to use the `HideAfterFirstScroll` option for fixed or sticky position elements such as sticky headers or consent banners.
+:::
+
+Options:
+
+- `DelayAfterScrollMs`: Delay in ms after scrolling and before taking screenshots (helps with lazy loading content)
```

Comment:
What is the default value?

Reply:
there is no default value so no value == no delay

Reply:
Should we mention the default behaviour ?

Reply:
IMO no need.
I would expect it to be 0 if nothing was mention in the docs. What about you guys?